templates/bird.conf.j2

```diff
@@ -0,0 +1,18 @@
+{% macro print_config(config) %}
+{% if config is string %}
+{{ config }}
+{%- elif config is mapping %}
+{% for key, value in config.items() %}
+{{ key }} {
+    {{ print_config(value) | indent }}}
+{%- endfor %}{# for key, value in element.items() #}
```

Comment:
Why this commented-out `for` header?

Reply:
Such that you know which loop this `endfor` belongs to.

Reply:
In that case:
```suggestion
{%- endfor %}{# for key, value in config.items() #}
```